.github/workflows/build.yml

```diff
@@ -26,6 +26,6 @@ jobs:
       - name: Install dependencies
         run: pnpm install
       - run: pnpm run lint
-      - run: pnpm run prettier . --check
+      - run: pnpm prettier . --check
```

Comment:
Using `pnpm run` implies use of a `package.json` script - whereas in this context we want the _binary_ (we don't want to run `prettier . --write`).